src/sx-design/src/Image/Image.js

```diff
@@ -76,18 +72,15 @@ const styles = sx.create({
   background: {
     display: 'flex',
     flexDirection: 'column',
-  },
-  backgroundLight: {
-    backgroundColor: 'lightgrey',
-  },
-  backgroundDark: {
-    backgroundColor: 'grey',
+    backgroundColor: 'rgba(var(--sx-accent-2))',
```

Comment:
This is why it's important to prefer colors from SX Design instead of re-inventing the wheel: `--sx-accent-2` is already light/dark theme sensitive and there is no need for manual custom colors based on the theme. It makes the component implementation much nicer.